src/main/webapp/app/course/manage/course-management.component.ts

```diff
@@ -24,6 +25,8 @@ export class CourseManagementComponent implements OnInit, OnDestroy, AfterViewIn
     showExamButton = true;
 
     courses: Course[];
+    courseSemesters: string[];
+    semesterCollapsed: object;
```

Comment:
Any reason for this just being a generic object? Maybe we could write a type here? Not really needed just a suggestion :)

Reply:
I wasn't sure what to put as type there, so I oriented this on what was done in https://github.com/ls1intum/Artemis/blob/develop/src/main/webapp/app/overview/course-exercises/course-exercises.component.ts#L46

Reply:
This should do:
```suggestion
    semesterCollapsed: { [key:string]: boolean; };
```

OR: Consider using the TypeScript `Map` instead.